distribution/src/main/resources/bin/elasticsearch

```diff
@@ -14,21 +14,20 @@
 #
 #   ES_JAVA_OPTS="-Xms8g -Xmx8g" ./bin/elasticsearch
 
+source "`dirname "$0"`"/elasticsearch-env
+
 parse_jvm_options() {
   if [ -f "$1" ]; then
-    echo "$(grep "^-" "$1" | tr '\n' ' ')"
+    echo "`grep "^-" "$1" | tr '\n' ' '`"
```

Comment:
Oh, and to be clear, it's to backticks for now because there are fewer that are using `$()` then backticks so the smallest path to consistency is to backticks. Then I'll open a separate issue sometime a few weeks from now to go to all `$()`.